CONTRIBUTING.md

```diff
@@ -2,5 +2,5 @@
 
 At Reaction Commerce, we're dedicated to the open source community. In fact, we've designed our entire platform and business to grow from the passion and creativity that an open source community ignites. We've already attracted a small, dedicated team of open source contributors, and there's always room for more.
 
-If you'd like to join us, check out our detailed [Contributing Guide](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction).
+If you'd like to join us, let's meet in Discord [Developer Chat](https://discord.gg/Bwm63tBcQY).
```

Comment:
I think we shouldn't have the Gitter link here. It should be some kind of contributing guide.

Reply:
yeah, that page is pending (working on that tomorrow), should we wait for this merge until that page is ready?

Reply:
this is fixed